gdb/plan.sh

```diff
@@ -1,32 +1,55 @@
 pkg_name=gdb
 pkg_origin=core
-pkg_version=7.11
+pkg_version=7.11.1
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('gplv3')
-pkg_source=http://ftp.gnu.org/gnu/$pkg_name/${pkg_name}-${pkg_version}.tar.xz
-pkg_shasum=7a434116cb630d77bb40776e8f5d3937bed11dea56bafebb4d2bc5dd389fe5c1
-pkg_deps=(core/glibc core/readline core/zlib core/python)
-pkg_build_deps=(core/coreutils core/diffutils core/patch core/make core/gcc core/texinfo)
+pkg_source=http://ftp.gnu.org/gnu/${pkg_name}/${pkg_name}-${pkg_version}.tar.xz
+pkg_shasum=e9216da4e3755e9f414c1aa0026b626251dfc57ffe572a266e98da4f6988fc70
+pkg_deps=(core/glibc core/readline core/zlib core/expat core/python)
+pkg_build_deps=(core/coreutils core/diffutils core/expect core/dejagnu core/patch core/make core/gcc core/texinfo)
 pkg_bin_dirs=(bin)
 pkg_lib_dirs=(lib)
 
 do_build() {
+  CFLAGS="${CFLAGS} -O2 -fstack-protector-strong -Wformat -Werror=format-security "
+  CXXFLAGS="${CXXFLAGS} -O2 -fstack-protector-strong -Wformat -Werror=format-security "
+  CPPFLAGS="${CPPFLAGS} -Wdate-time"
+  LDFLAGS="${LDFLAGS} -Wl,-Bsymbolic-functions -Wl,-z,relro"
+
   ./configure \
-    --prefix=$pkg_prefix \
+    --build=x86_64-linux-gnu \
+    --host=x86_64-linux-gnu \
+    --prefix="${pkg_prefix}" \
+    --sysconfdir=/etc \
+    --localstatedir=/var \
+    --libexecdir="${prefix}/lib/gdb" \
+    --disable-maintainer-mode \
+    --disable-dependency-tracking \
+    --disable-silent-rules \
+    --disable-gdbtk \
+    --disable-shared \
+    --with-pkgversion="The Habitat Maintainers ${pkg_version}/${pkg_release}" \
     --with-system-readline \
-    --with-system-zlib
-  make
+    --with-system-zlib \
+    --with-expat \
+    --without-lzma \
+    --without-guile \
```

Comment:
I looked at `Guile` and not many use it(also, Ubuntu disables it).
But why not. Btw, I would like more `babeltrace` to be enabled, but there is no package at the moment(and Ubuntu disables this one too).

Would you like me to try to enable `Guile`?